test/acceptance/features/steps/openshift.py

```diff
@@ -185,6 +185,7 @@ def apply(self, yaml, namespace=None):
         return output
 
     def apply_invalid(self, yaml, namespace=None):
+        print("AAAAAAAAAAAA", yaml)
```

Comment:
this should be removed, right?

Reply:
Removed. c46fcb9aa05b6bd0887a6a8dab12615d9fc47c1f